script/Ciel/Day 5/05_01_CIEL05_4.txt

```diff
@@ -964,7 +964,7 @@ The Principle has yet to take root.
 [sha:aba520a412339e7ce1a38c7198a3aecd16231d8f]{
 -- Page 729, Offset 12827.
 -- 「“――――――貴様”」
-"――――――You!"
+"―――You!"
```

Comment:
Same as last comment, I guess. You!! and stuff like that is very much JPisms, I guess. I have this vague memory of us talking about this line, but I can't remember many details.

Reply:
Knave?

Reply:
At the very least, just change to "you bastard" like it is later in the file; it's not perfect, but it's far preferrable.